Civi/API/Api3SelectQuery.php

```diff
@@ -106,7 +106,7 @@ protected function buildWhereClause() {
         continue;
       }
       $operator = is_array($value) ? \CRM_Utils_Array::first(array_keys($value)) : NULL;
-      if (!in_array($operator, \CRM_Core_DAO::acceptedSQLOperators())) {
+      if (!in_array($operator, \CRM_Core_DAO::acceptedSQLOperators(), TRUE)) {
```

Comment:
It's a precaution. As I said in the commit message, there's a bug in php (at least some versions) where the function incorrectly returns `true` if the first param is `0`.